lib/Conversion/TorchToTMTensor/TorchToTMTensor.cpp

```diff
@@ -523,6 +548,50 @@ class ConvertAtenMaxPool2dWithIndicesBackwardOp
 };
 } // namespace
 
+namespace {
+class ConvertAtenCumsumOp : public OpConversionPattern<AtenCumsumOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(AtenCumsumOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+
+    Value input = adaptor.self();
+    auto resultType = input.getType().cast<RankedTensorType>();
+    Type elementType = resultType.getElementType();
+    Location loc = op->getLoc();
+    SmallVector<Value> sizes = getTensorSizes(rewriter, loc, input);
+    Value output = createZeroInitTensor(rewriter, loc, sizes, elementType);
+    output = rewriter.create<tensor::CastOp>(loc, resultType, output);
+
+    int64_t dim;
+    if (!matchPattern(op.dim(), m_TorchConstantInt(&dim)))
+      return rewriter.notifyMatchFailure(
+          op, "unimplemented: only constant dim value is supported");
+
+    sizes.erase(sizes.begin() + dim);
```

Comment:
nit: I think it would improve readability to create a new vector `SmallVector<Value> accSizes(sizes)`, and erase the element from that vector.